src/dashboard/src/components/file/views.py

```diff
@@ -0,0 +1,210 @@
+# Standard library, alphabetical by import source
+import json
+import logging
+import os
+import sys
+
+# Django Core, alphabetical by import source
+
+# External dependencies, alphabetical
+import requests
+
+# This project, alphabetical by import source
+from components import helpers
+from main import models
+
+sys.path.append("/usr/lib/archivematica/archivematicaCommon")
+import elasticSearchFunctions
+import storageService as storage_service
+
+logger = logging.getLogger('archivematica.dashboard')
+
+def transfer_file_tags(request, uuid):
+    """
+    Fetch, set or remove tags from a record.
+    All methods return 404 if a record can't be retrieved for the given UUID, and 400 if more than one file was found for a given UUID.
+
+    Accepted methods and parameters:
+        GET: requires a file UUID.
+            Returns a JSON-encoded list of the file's tags on success.
+        PUT: requires a file UUID, and document body must be a JSON-encoded list.
+            Replaces the list of tags in the record with the provided list.
+            Returns {"success": true} on success.
+            Returns 400 if no JSON document is provided in the request body, if the body can't be decoded, or if the body is any JSON object other than a list.
+        DELETE: requiest a file UUID.
+            Deletes all tags for the given file.
+    """
+    if request.method == 'GET':
```

Comment:
This may be a good fit for Django's [class-based views](https://docs.djangoproject.com/en/dev/topics/class-based-views/) since they have different functions for different verbs.  Take a look at it?